Components/Toast/toast.css

```diff
@@ -0,0 +1,42 @@
+
+.element-display {
+    display: flex;
+    justify-content: center;
+    flex-direction: row;
```

Comment:
default direction is row so no need to set

Reply:
okay, Thanks